back/CourseSheetModule/CourseSheetMod.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
```

Comment:
Doesn't respect the classes diagram
    CourseSheetModel is an attribute
    the class should be named CourseSheetModule, not CourseSheetMod
    it has to take an array in parameter, not take $_GET